### ISSUE TITLE
generate 的时候出现 The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results. Setting `pad_token_id` to `eos_token_id`:151643 for open-end generation. 这是什么意思，有什么影响？

### ISSUE THREAD

**Issue by zhiguoxu** [QwenLM/Qwen1.5#26]

**Comment by JustinLin610** [comment]
No need to worry about it. As we do not have default pad_token, it will automatically use the eos token for padding

**Comment by turkeymz** [comment]
> No need to worry about it. As we do not have default pad_token, it will automatically use the eos token for padding

how should I  ignore this log?

**Comment by BeyondTheStarry** [comment]
if you don't want that log ，You can refer to the following suggestions.
......
#model _ input has the attention _ mask attribute.。
**attention_mask = model_inputs['attention_mask']**
......
#Supplementary boldface parameter
generated_ids = model.generate(
  model_inputs.input_ids,
  **attention_mask=attention_mask,**
  max_new_tokens=400,
  repetition_penalty=1.15,
  **pad_token_id=tokenizer.eos_token_id**
)

**Comment by VictoireWood** [comment]
> if you don't want that log ，You can refer to the following suggestions. ...... #model _ input has the attention _ mask attribute.。 **attention_mask = model_inputs['attention_mask']** ...... #Supplementary boldface parameter generated_ids = model.generate( model_inputs.input_ids, **attention_mask=attention_mask,** max_new_tokens=400, repetition_penalty=1.15, **pad_token_id=tokenizer.eos_token_id** )

What if I'm runing the code from readme:
```
from transformers import AutoModelForCausalLM, AutoTokenizer
from transformers.generation import GenerationConfig
import torch
import os
import pandas as pd
from glob import glob
from tqdm import tqdm, trange
import warnings
warnings.filterwarnings("ignore")

torch.manual_seed(1234)

# Note: The default behavior now has injection attack prevention off.
tokenizer = AutoTokenizer.from_pretrained("/root/workspace/VL/Qwen-VL-Chat", trust_remote_code=True)

# use bf16
# model = AutoModelForCausalLM.from_pretrained("Qwen/Qwen-VL-Chat", device_map="auto", trust_remote_code=True, bf16=True).eval()
# use fp16
# model = AutoModelForCausalLM.from_pretrained("Qwen/Qwen-VL-Chat", device_map="auto", trust_remote_code=True, fp16=True).eval()
# use cpu only
# model = AutoModelForCausalLM.from_pretrained("Qwen/Qwen-VL-Chat", device_map="cpu", trust_remote_code=True).eval()
# use cuda device
model = AutoModelForCausalLM.from_pretrained("/root/workspace/VL/Qwen-VL-Chat", device_map="cuda", trust_remote_code=True).eval()

# Specify hyperparameters for generation
model.generation_config = GenerationConfig.from_pretrained("/root/workspace/VL/Qwen-VL-Chat", trust_remote_code=True)

# 1st dialogue turn
query = tokenizer.from_list_format([
    {'image': 'https://qianwen-res.oss-cn-beijing.aliyuncs.com/Qwen-VL/assets/demo.jpeg'}, # Either a local path or an url
    {'text': '这是什么?'},
])
response, history = model.chat(tokenizer, query=query, history=None)
print(response)
```
Where should be changed to stop that warning?
